sql/2021/pwa/top_manifest_categories.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL
+# Top manifest categories
```

Comment:
This is one of those things that sound like great information, for example, to understand which verticals use or benefit more from PWAs. Despite that, I'm still unsure about what's the real usefulness of it (I'll ask).
Regarding the query itself, besides the distribution, it would be useful to know the % of sites that, having a manifest, have this field.

Reply:
The query already does that. Running it for June for example we have:

![image](https://user-images.githubusercontent.com/10931297/126152023-db47a669-a858-4b10-8f60-b0d9dde41f13.png)

So 252 of the 2603 desktop sites that have a manifest use the News category.
And 150 of the 2603 desktop sites use the Education category.
As you can see in both cases it's measured against the total of 2603 manifest sites.

Remember that we only got the tail end of the June run so these numbers will be small.

Reply:
Perfect, thank you!